### ISSUE TITLE
Can you add .ics file type?

### ISSUE THREAD

**Issue by Skylaunch** [crazecoder/open_file#306]
I need to open the .ics file. 

When opening with the command "await Open File.open(file.path);", all possible applications are shown to me. 

However, when opening the saved file manually, the correct Outlook is displayed.

It would be great to offer to open the file via Outlook if it is installed

**Comment by crazecoder** [comment]
The choice of which app to open is the user's personal habit, which should not interfere with the user's choice. If the user needs to open an app directly next time, they can select the "always" option when opening it for the first time

**Comment by Skylaunch** [comment]
Yes, you right, but how can I reduce the count of applications that cannot physically open the ics file?

**Comment by crazecoder** [comment]
On Android devices, it will look for apps that are configured to support it in the manifest. On iOS, it will list all apps that can be opened or transferred. These are controlled by the system. I have not found a way to modify them yet.

**Comment by adrianwinau** [comment]
Specify the type to reduce the shown apps: OpenFile.open("event.ics", type: "text/calendar")
